script.js

```diff
@@ -0,0 +1,50 @@
+var todoList = new Array();
+var ul=document.getElementById('todoList');
+function addList() {
+	
+	var inputValue = document.getElementById('todoInput').value;
```

Comment:
Use proper naming conventions for a variable if you are showing it as a task the then name it as a task.  

For multiple tasks, You should use tasks or taskList like that